broke-broker/src/main/java/org/dsngroup/broke/broker/channel/handler/MqttMessageHandler.java

```diff
@@ -0,0 +1,67 @@
+package org.dsngroup.broke.broker.channel.handler;
+
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelInboundHandlerAdapter;
+import io.netty.handler.codec.mqtt.*;
+import org.dsngroup.broke.protocol.ConnectMessage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MqttMessageHandler extends ChannelInboundHandlerAdapter{
+
+    private final static Logger logger = LoggerFactory.getLogger(MqttMessageHandler.class);
+
+    private boolean isConnected = false;
+
```

Comment:
The connected client’s information should be stored at some place? It’s not proper check here.